src/modules/client/components/clientAlerts/CreateClientAlertDialog.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Alert,
+  Box,
+  DialogContent,
+  DialogProps,
+  DialogTitle,
+} from '@mui/material';
+import CommonDialog from '@/components/elements/CommonDialog';
+import StaticForm from '@/modules/form/components/StaticForm';
+import { clientBriefName } from '@/modules/hmis/hmisUtil';
+import {
+  ClientAlertInput,
+  ClientWithAlertFieldsFragment,
+  CreateClientAlertDocument,
+  CreateClientAlertMutation,
+  CreateClientAlertMutationVariables,
+  StaticFormRole,
+} from '@/types/gqlTypes';
+
+interface ClientAlertDialogProps extends DialogProps {
+  client: ClientWithAlertFieldsFragment;
+}
+export const CreateClientAlertDialog: React.FC<ClientAlertDialogProps> = ({
+  client,
+  ...props
+}) => {
+  const clientName = clientBriefName(client);
+
+  return (
+    <CommonDialog maxWidth='sm' fullWidth {...props}>
+      <DialogTitle>Create Alert for {clientName}</DialogTitle>
+      <DialogContent>
+        <Box mt={2}>
+          <Alert sx={{ mb: 2 }} severity='info'>
+            Reminder: All staff who can view {clientName} will be able to see
+            this alert
+          </Alert>
+          <StaticForm<
+            CreateClientAlertMutation,
+            CreateClientAlertMutationVariables
+          >
+            role={StaticFormRole.ClientAlert}
+            mutationDocument={CreateClientAlertDocument}
+            getVariables={(values) => {
+              return {
+                input: {
+                  clientId: client.id,
+                  ...values,
+                } as ClientAlertInput,
+              };
+            }}
+            getErrors={(data) => data.createClientAlert?.errors || []}
+            onCompleted={(data) => {
+              if (!data?.createClientAlert?.errors?.length && props.onClose)
+                props.onClose({}, 'escapeKeyDown');
+            }}
```

Comment:
Also, here we need to re-fetch the client so that the newly created alert shows up right away. It looks like Apollo has some built-in functionality for this available in react hooks but I wasn't quite understanding how to incorporate it; maybe we can pair on this tomorrow?

Reply:
Two other topics I'd like to pair on tomorrow:
- I've picked 'CHOICE' as the type for the priority field [here](https://github.com/greenriver/hmis-warehouse/pull/3952/files#diff-cd3fe7155a7e32a7811cb8434c97e51b7f2aa8694788e5a1c72e9516720fb1ec), and it shows up as radio buttons, whereas other 'CHOICE' types I'm seeing in the json form builder show up as dropdowns -- is this because there are only 3 options? (also, Ash indicated we should add colored circles next to the priorities in the dropdown, but since it is a dynamically built form, I am not sure how to do this)
- The `priority` and `note` fields should be required, and in the json definition linked above they are, but it's not taking effect in the same way that it does for AutoExit - so I'd like to investigate this together and figure out which part I am missing